### DISCUSSION TITLE
Does anyone using Prune function? And is it work?

### DISCUSSION THREAD

**Discussion by ngocnd1** [bb-Ricardo/netbox-sync#310] [General]
Hi anyone, I had turned on prune function to delete component that already tag orphaned, settings.ini like below.
    prune_enabled = True
    prune_delay_in_days = 1
By the way i configured syncing daily, in netbox dashboard already shown it will be delete in 1 days but it's doesnt work.
Anyone can help?
Thanks if any.

**Comment by muratbulat** [comment]
Hi @ngocnd1 
I tested, this function working. Can you share debug log for prune outputs?

**Reply by ngocnd1** [reply]
Hi @muratbulat 
Prune output is just
Changed from a source which has previously provided this object no longer states it exists. Prune is disable to Pruning is enabled and objects will be automatically removed after 1 days,
I didnt see any delete logs.

**Reply by bb-Ricardo** [reply]
any updates on this? what happens if you set `prune_delay_in_days` to `0`?
